src/openrct2/ReplayManager.cpp

```diff
@@ -768,11 +768,11 @@ namespace OpenRCT2
                 }
 
                 // Focus camera on event.
-                if (isPositionValid && gCommandPosition.x != LOCATION_NULL)
+                if (isPositionValid && result->Position.x != LOCATION_NULL)
```

Comment:
Shouldn't we just call `isNull` on the coords?

Reply:
No it would be incorrect that checks for coords null this is location null. Also isnull is a function of coordsxyzd this is coordsxyz.

Reply:
> No it would be incorrect that checks for coords null this is location null.

Why do we have two ways of marking null coords anyway?

> Also isnull is a function of coordsxyzd this is coordsxyz.

Then it should be made available on CoordsXY and CoordsXYZ. There is no reason why it should be CoordsXYZD-only.

Reply:
While I agree I don't think I will do it in this PR. I need to work out what functions are using which version of null before doing that and it will not be pretty until it is resolved.

Reply:
That is okay.